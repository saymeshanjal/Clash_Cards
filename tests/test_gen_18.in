21
14 83 1
6 148 3
6 89 1
12 70 3
2 246 2
12 192 2
10 111 2
12 239 3
4 83 4
14 149 5
10 235 2
8 69 2
12 129 3
16 15 2
6 195 3
6 110 5
10 185 4
2 227 2
10 15 1
14 215 4
14 17 3
