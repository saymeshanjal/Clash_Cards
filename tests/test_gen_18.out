1716 1716 0
6 14 0
10 10 0
430 494 0
0 1 1
286 287 351
76 77 99
430 494 0
0 5 1
987 2445 0
76 77 99
20 21 29
430 494 0
603 2417 9850
6 14 0
1 19 0
44 133 75
0 1 1
116 136 0
910 1520 1002
435 2997 0
